### PR TITLE
2381 ues takeover

### LINKED ISSUE #2381: UES takeover build

### PR DESCRIPTION
**Pull request by grahambancroft**
## Done

Add new UES takeover

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Make sure the takeover correlates to the [copy doc](https://docs.google.com/document/d/1yjUvj-B_7KeAvV3HlNnnq45Ndc1SuK8Q1bwOZcgBVUs/edit) and [design](https://app.zenhub.com/workspace/o/canonical-websites/www.ubuntu.com/issues/2381) 

## Issue / Card

Fixes #2381

### CODE REVIEW

**Comment by anthonydillon** [comment]
@grahambancroft why have you set +1 and -1 labels to this PR? If it has been reviewed by UX and design please mention that and by who. Also if it is not ready for code and QA then please leave the labels off for now and just leave the don't merge label on it.

**Comment by grahambancroft** [comment]
@anthonydillon UX +1 was @frankiegranato, design +1 was @michaelb17 
I've taken the other labels off for now, I still want to make some code changes these amends won't change the design.

**Comment by anthonydillon** [comment]
@grahambancroft thanks. A PR shouldn't be reviewed unless it has needed labels.

File: templates/takeovers/_ues_takeover.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<section class="p-strip--image is-dark is-deep p-takeover" style="background-image: url('{{ ASSET_SERVER_URL }}ebdfffbf-Aubergine_suru_background.png'); background-position: 77% 0%;">
+  <div class="row">
+    <div class="u-equal-height u-vertically-center">
+      <div class="col-6" itemscope itemtype="https://schema.org/Event">
+        <h1 class="p-heading--two">Find out how the world&rsquo;s top companies use Ubuntu to succeed</h1>
+        <p class="p-heading--five"><span class="event-title" itemprop="name">Ubuntu Enterprise Summit</span>, <span class="event-date" itemprop="startDate" content="2017-12-05T16:00">December 5th</span> &ndash; <span class="event-date" itemprop="endDate" content="2017-12-06T21:00">6th</span></p>
```
Comment (pmahnke) [inline templates/takeovers/_ues_takeover.html]: channeling my inner @yaili  - should be '5-6 December 2017'

File: templates/takeovers/_ues_takeover.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<section class="p-strip--image is-dark is-deep p-takeover" style="background-image: url('{{ ASSET_SERVER_URL }}ebdfffbf-Aubergine_suru_background.png'); background-position: 77% 0%;">
+  <div class="row">
+    <div class="u-equal-height u-vertically-center">
+      <div class="col-6" itemscope itemtype="https://schema.org/Event">
+        <h1 class="p-heading--two">Find out how the world&rsquo;s top companies use Ubuntu to succeed</h1>
```
Comment (pmahnke) [inline templates/takeovers/_ues_takeover.html]: I think 'p-heading--three' looks better

image

**Comment by pmahnke** [comment]
I would change the order of the text around

```
<div class="col-6" itemscope="" itemtype="https://schema.org/Event">
  <h1 class="p-heading--two"><span class="event-title" itemprop="name">Ubuntu Enterprise Summit</span></h1>
  <h2 class="p-heading--three"><span class="event-date" itemprop="startDate" content="2017-12-05T16:00">5</span> – <span class="event-date" itemprop="endDate" content="2017-12-06T21:00">6 December 2017</span></h2>
  <p class="p-heading--five">Find out how the world’s top companies use Ubuntu to succeed</p>
  <p class="u-hide--small u-show--medium u-show--large"><a itemprop="url" href="https://ubuntu.brighttalk.com/summit/ubuntu-enterprise-summit/" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : 'UES takeover', 'eventLabel' : 'Sign up now', 'eventValue' : undefined });"
      class="p-button--positive">Sign up now</a></p>
</div>
```

image

**Comment by grahambancroft** [comment]
Done

**Pull request merged**